swupd/delta_test.go

```diff
@@ -0,0 +1,26 @@
+package swupd
+
+import (
+	"path/filepath"
+	"testing"
+)
+
+var largeString = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque tempor quam id convallis convallis. Proin vehicula nisi in augue posuere lobortis. Quisque tincidunt elit ac facilisis auctor. Praesent facilisis ex eros, nec blandit dui suscipit porta. Nunc id nunc rhoncus, condimentum nibh at, fermentum sem. Nam mollis justo ac iaculis gravida. Vestibulum convallis congue dolor, vitae rutrum ex finibus vel. Phasellus convallis sem nunc, ac laoreet risus rhoncus fermentum. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nullam eget pellentesque nulla. Etiam tristique non magna quis consectetur. In ac dolor sagittis, vulputate ante tincidunt, suscipit leo."
```

Comment:
Another nitpick:

Can you line-wrap this? I just think it looks better in editors to have something like
```go
var largeString = `something really long that
spans multiple lines instead of one really long
line that some editors treat differently than
others`
```

Reply:
So you can read the latin more easily? *puts cheeky comment away* :grinning:

Reply:
Who says I can't read latin?

Reply:
As someone who studied latin at school for 3 years before I was allowed to drop it, I prefer `something really long that...' as it makes sense, unlike random text designed to show off typefaces. Heck even `A quick brown fox jumps over the lazy dog` and related [pangrams](https://en.wikipedia.org/wiki/Pangram) make more sense.

Of course it wasn't all wasted, it makes me appreciate [Asterix)[https://en.wikipedia.org/wiki/List_of_Asterix_volumes] more.

Reply:
Hrm being able to enjoy Asterix more than I have already, I'm unable to imagine such a fine world.